pkg/testmachinery/types.go

```diff
@@ -26,6 +26,9 @@ const (
 	// TM_KUBECONFIG_PATH is the path where kubeconfigs are mounted to tests.
 	TM_KUBECONFIG_PATH = "/tmp/env/kubeconfig"
 
+	// TM_KUBECONFIG_PATH is the path where kubeconfigs are mounted to tests.
+	TM_SHARED_PATH = "/tmp/shared"
```

Comment:
I would change this to a testmachinery namespace --> `tmp/tm/shared`

Reply:
don't know why we had TM_KUBECONFIG_PATH at `tmp/env/kubeconfig`, maybe you could change this path as well to `tmp/tm/kubeconfig`

Reply:
Done